lib/client.js

```diff
@@ -158,6 +158,123 @@ class Client {
     const relativePathWithQuery = `${params.resourceType}?${new URLSearchParams(params.searchParams)}`;
     return this.httpClient.get(relativePathWithQuery);
   }
+
+  /**
+   * Create a resource by FHIR id.
+   *
+   * @example
+   * const newPatient = { resourceType: 'Patient',
+   *                      active: true,
+   *                      name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+   *                      gender: 'female',
+   *                      birthDate: '1948-04-14',
+   *                    }
+   *
+   * // Using promises
+   * fhirClient.create({ resourceType: 'Patient',
+   *                     resource: newPatient,
+   *                   })
+   *           .then((data) => { console.log(data); });
+   *
+   * // Using async
+   * let response = await fhirClient.create({ resourceType: 'Patient',
+   *                                          resource: newPatient,
+   *                                         })
+   * console.log(response);
+   *
+   * @param {Object} params - The request parameters.
+   * @param {string} params.resource - The new resource data to create.
+   *
+   * @return {Promise<Object>} FHIR resource
+   */
+  create(params) {
+    return this.httpClient.post(params.resourceType, params.resource);
+  }
+
+  /**
+   * Delete a resource by FHIR id.
+   *
+   * @example
+   *
+   * // Using promises
+   * fhirClient.delete({ resourceType: 'Patient', id: 12345 })
+   *           .then((data) => { console.log(data); });
+   *
+   * // Using async
+   * let response = await fhirClient.delete({ resourceType: 'Patient', id: 12345 });
+   * console.log(response);
+   *
+   * @param {Object} params - The request parameters.
```

Comment:
Technically, is this actually a "FHIR resource" -- or somesuch?

...and not "request parameters" ?

e.g.: this is the JSON *body* / request payload?

Reply:
This is not a FHIR resource.

Reply:
Is it "FHIR JSON payload"  ?

Reply:
My questions are:

* whatever it is, it is still a repeating *type* of some sort?

* it's actually *not* a set of "request params", as described?

Reply:
(we already use the terminology "request params" for URL-query-params)